test/auth/unified/reauthenticate_without_retry.json

```diff
@@ -0,0 +1,201 @@
+{
+  "description":"reauthenticate_without_retry",
+  "schemaVersion":"1.12",
+  "runOnRequirements":[
+    {
+      "minServerVersion":"6.3",
+      "auth":true
+    }
+  ],
+  "createEntities":[
+    {
+      "client":{
+        "id":"client0",
+        "uriOptions":{
+          "retryReads":false,
+          "retryWrites":false
+        },
+        "observeEvents":[
+          "commandStartedEvent",
+          "commandSucceededEvent",
+          "commandFailedEvent"
+        ]
+      }
+    },
+    {
+      "database":{
+        "id":"database0",
+        "client":"client0",
+        "databaseName":"db"
+      }
+    },
+    {
+      "collection":{
+        "id":"collection0",
+        "database":"database0",
+        "collectionName":"collName"
+      }
+    }
+  ],
+  "initialData":[
+    {
+      "collectionName":"collName",
+      "databaseName":"db",
+      "documents":[
+
+      ]
+    }
+  ],
+  "tests":[
+    {
+      "description":"Read command should reauthenticate when receive ReauthenticationRequired error code and retryReads=false",
+      "operations":[
+        {
+          "name":"failPoint",
+          "object":"testRunner",
+          "arguments":{
+            "client":"client0",
+            "failPoint":{
+              "configureFailPoint":"failCommand",
+              "mode":{
+                "times":1
+              },
+              "data":{
+                "failCommands":[
+                  "find"
+                ],
+                "errorCode":391
+              }
+            }
+          }
+        },
+        {
+          "name":"find",
+          "arguments":{
+            "filter":{
+
+            }
+          },
+          "object":"collection0",
+          "expectResult":[
+
+          ]
+        }
+      ],
+      "expectEvents":[
+        {
+          "client":"client0",
+          "events":[
+            {
+              "commandStartedEvent":{
+                "command":{
+                  "find":"collName",
+                  "filter":{
```

Comment:
What's up with the formatting here? It looks like these json files are different from the ones in the spec repo.

Reply:
Fixed